content/integrations/awsdirectconnect.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
+
+Enable this integration to see in Datadog all your Direct Connect metrics
```

Comment:
Can we name a few examples of useful metrics here? Also, our integration overviews don't need to tell readers to enable the integration. Let's just say "This integration collects <example metrics/events/checks> from AWS Direct Connect".